client/components/social-buttons/google.js

```diff
@@ -101,6 +101,10 @@ class GoogleLoginButton extends Component {
 		} );
 
 		this.setState( { isDisabled: false } );
+
+		if ( this.props.launchServiceOnLoad ) {
```

Comment:
Should we log another event here, something like `calypso_social_button_launched_on_load`?